pkg/manager/router/router_score.go

```diff
@@ -408,6 +493,7 @@ func (router *ScoreBasedRouter) ConnCount() int {
 		backend := be.Value
 		j += backend.connList.Len()
 	}
+	j += router.pausedConnList.Len()
```

Comment:
If `enablePause=false`, `pausedConnList` is nil. You can just always create `pausedConnList`.